plugins/stats_table_procedure.cc

```diff
@@ -110,18 +122,18 @@ save(const std::string & filename) const
 void StatsTable::
 serialize(ML::DB::Store_Writer & store) const
 {
-    int version = 1;
+    int version = 2;
     store << version << colName << outcome_names << counts << zeroCounts;
 }
 
 void StatsTable::
 reconstitute(ML::DB::Store_Reader & store)
 {
     int version;
-    int REQUIRED_V = 1;
+    int REQUIRED_V = 2;
     store >> version;
     if(version!=REQUIRED_V) {
-        throw ML::Exception(ML::format(
+        throw HttpReturnException(400, ML::format(
```

Comment:
we can sacrifice back-compatibility of model files for stats tables today. Anyone using them will need to migrate a bunch of code anyway to track this change, and we will help.